packages/react-core/src/components/TreeView/TreeViewRoot.tsx

```diff
@@ -55,8 +73,8 @@ export class TreeViewRoot extends React.Component<TreeViewRootProps> {
       el => !el.classList.contains('pf-m-disabled')
     );
 
-    if (key === 'Space') {
-      (document.activeElement as HTMLElement).click();
+    if (key === ' ') {
```

Comment:
Nit: could we import the KeyTypes constant and use that here (and on line 137) instead of using a hardcoded `' '`?

Reply:
Sure thing